docs/McpClient-Feedback.md

```diff
@@ -0,0 +1,47 @@
+ModelContextProtocol.Protocol.Implementation name is too generic.  Lots of things in this namespace named poorly.  That might be OK if users don't have to deal with them, but it looks like they do.
```

Comment:
Per discussion - Protocol types will not be designed - instead we'll leave them to match the MCP spec types and avoid exposing them for mainline scenarios.